apps/web/src/components/sidebar/SafeListContextMenu/index.tsx

```diff
@@ -78,6 +90,20 @@ const SafeListContextMenu = ({
         <MoreVertIcon sx={({ palette }) => ({ color: palette.border.main })} />
       </IconButton>
       <ContextMenu anchorEl={anchorEl} open={!!anchorEl} onClose={handleCloseContextMenu}>
+        {!undeployedSafe && nestedSafes?.safes && nestedSafes.safes.length > 0 && (
```

Comment:
```suggestion
        {!undeployedSafe && nestedSafes?.safes?.length > 0 && (
```

Reply:
TypeScript doesn't like that:

![image](https://github.com/user-attachments/assets/43ab3ad6-e089-490f-ae87-daaa07a77ed2)